packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -675,6 +675,12 @@ export const audiusBackend = ({
       monitoringCallbacks.contentNode
     )
 
+    const useDiscoveryRelay = await getFeatureEnabled(
+      FeatureFlags.DISCOVERY_RELAY
+    )
+
+    console.log({ useDiscoveryRelay }, `discovery relay${useDiscoveryRelay ? ' ' : ' not '}enabled`)
```

Comment:
If this is intentional can you make it a console.info?

Reply:
yes! forgot that